content/blog/zenodo-community.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "New Zenodo Openscapes Community helps you Find and Cite Openscapes things"
+author: Stefanie Butland
+date: '2023-05-31'
+slug: zenodo-community
+categories:
+  - how-we-work
+  - noaa-fisheries
+  - nasa-framework
+tags:
+  - impact
+  - noaa-fisheries
+  - nasa-framework
+banner: img/blog/zenodo-openscapes-community-square.png
+---
+
+# We created a Zenodo Openscapes Community!
+
+We want to share our work in a way that people can find it, use it, improve it, and cite it, or get credit for their contributions. For people who have participated in our programs like Openscapes Champions, or Pathways to Open Science, we want a robust way to add these to their CV as professional development. For contributors to our open educational resources, like the NASA Earthdata Cloud Cookbook, we want them to get credit and visibility. For people who want to reuse or remix a slide from a presentation, we want them to feel great about it by providing an easy way to cite the presentation. To enable all of this, we created a [Zenodo Openscapes Community](https://zenodo.org/communities/openscapes/) as a semantically meaningful group of selected research products. NASA's Transform to Open Science ([TOPS](https://zenodo.org/communities/tops/)) and the Center for Scientific Collaboration and Community Engagement's ([CSCCE](https://zenodo.org/communities/cscce/)) Zenodo Communities were inspirations for ours.
+
+[Zenodo](https://zenodo.org/) is a general-purpose open repository that allows researchers to deposit research related digital artefacts like research papers, data sets, research software, reports, lesson materials, and presentations. For each submission, a persistent digital object identifier ([DOI](https://en.wikipedia.org/wiki/Digital_object_identifier)) is minted, which makes the stored items easily citable (adapted from [Wikipedia](https://en.wikipedia.org/wiki/Zenodo)). Zenodo allows for versioning and we can preserve GitHub repositories in Zenodo too (GitHub itself is *not* a repository!).
+
+<br> <center><a><img src="/img/blog/zenodo-openscapes-community.png" width="800px" alt="screenshot of the Openscapes Community page on Zenodo"></a><figcaption>Openscapes Community page on Zenodo</figcaption></center> <br>
+
+## Our use cases
+
+The Openscapes Approach Guide gives instructions for our use cases, with pointers for things that might not be obvious to a first-time user.
+
+-   What do we curate in our Zenodo Openscapes Community? ([link](http://localhost:6939/approach/tooling.html#what-do-we-curate-in-our-zenodo-openscapes-community))
```

Comment:
Maybe just link the highest level Zenodo link (https://openscapes.github.io/approach-guide/approach/tooling.html#sec-zenodo) from the first sentence just previous here, rather than all the [link]s

Reply:
haha that was my first thought before adding all links 😁. will do